libraries/testkit/sgaccel.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, cluster_config, target):
         self.ip = target["ip"]
         self.url = "http://{}:4985".format(target["ip"])
         self.hostname = target["name"]
+        self.cluster_config = cluster_config
+        self.server_port = 8091
+        self.server_scheme = "http"
```

Comment:
These are the default non-ssl values. It is also used in other methods like restart. I thought it is a good idea to initialize the defaults when the object is initialized